assemblies/assembly-optaplanner-school-timetable-quarkus.adoc

```diff
@@ -45,5 +45,34 @@ include::{enterprise-dir}/optimizer/business-optimizer-test-constraints-quarkus-
 include::{enterprise-dir}/optimizer/business-optimizer-test-solver-quarkus-proc.adoc[leveloffset=+2]
 include::{enterprise-dir}/optimizer/business-optimizer-logging-quarkus-proc.adoc[leveloffset=+1]
 include::{enterprise-dir}/optimizer/planner-quarkus-database-proc.adoc[leveloffset=+1]
+
+== Using Micrometer and Prometheus to monitor your school timetable OptaPlanner Quarkus application
+
+OptaPlanner exposes metrics through https://micrometer.io/[Micrometer], a metrics instrumentation library for Java applications. You can use Micrometer with Prometheus to monitor the OptaPlanner solver in the school timetable application.
+
+
+.Prerequisites
+* You have created the Quarkus OptaPlanner school timetable application.
```

Comment:
The same as here https://github.com/emmurphy1/kie-docs/pull/336/files#r710880591

Reply:
Updated

Reply:
LGTM

Reply:
LGTM